Added Test Case LG26

EverettHanke opened the pull request:
# Description

Implements the test case LG26.
Added the `expected_line_group_ri_15-20_mu_kW_gu_kW_st_2022-08-25%00#00#00_et_2022-10-24%00#00#00.csv` that was required in order for the test sample.

Assigned as a Codeday group project.
Group Members: @selenasat @MattMiss @EverettHanke

Partly addresses issue #962 Added tests for LG26

## Type of change

- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

selenasat committed:
added LG26

EverettHanke committed:
Added LG26 test cases

Co-authored-by: Matt Miss matt.w.miss@gmail.com
Co-authored-by: Everett Hanke evreeto@gmail.com
Co-authored-by: Selena Sat selena.ksat@gmail.com

EverettHanke committed:
Merge branch 'OpenEnergyDashboard:development' into development

Chocopepero commented:
Hello, thank you for your contribution to OED! All three contributors have signed the CLA, some last year and some recently.

The tests run as expected, and code looks good. (Minor issue that can be resolved by formatting the document).

huss committed:
minor change

This changed the test title which was wrong in the design doc.
It also formatted the file since it was off from before and continued.

huss commented:
For the record, I just pushed minor changes that were due to incorrect info in the design doc and formatting that was also present before. The test code in this PR was correct before this was done.

huss reviewed (approved):
Thanks to @selenasat, @MattMiss & @EverettHanke for this contribution to OED. Review and testing found it works as desired. Congratulations.
